src/components/MenuItem/MenuItem.jsx

```diff
@@ -18,7 +19,8 @@ export class MenuItem extends React.Component {
       isHot: !canBeHeated,
       total: price,
       modalOpen: false,
-      owner: ''
+      owner: '',
+      ownerId: cookie.load('ownerId')
```

Comment:
Its going to be loading the cookie for every menu item, can it be loaded once further up and then passed down?